eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp.hs

```diff
@@ -94,6 +94,7 @@ spec ::
   , Eq (Event (EraRule "ENACT" era))
   , Typeable (Event (EraRule "ENACT" era))
   , ToExpr (Event (EraRule "BBODY" era))
+  , EraPlutusTxInfo PlutusV1 era
   , EraPlutusTxInfo PlutusV2 era
```

Comment:
While you are at it you can move this constraint to `BabbageEraTest`
```suggestion
```

Reply:
I also added `EraPlutusTxInfo PlutusV3 era` to `ConwayEraTest`, while I was at it :+1: